src/ScottPlot4/ScottPlot/Plottable/ScatterPlot.cs

```diff
@@ -422,6 +423,30 @@ public void Render(PlotDimensions dims, Bitmap bmp, bool lowQuality = false)
                     DrawLinesWithGaps(points, gfx, penLine);
                 }
 
+                if (DataPointLabels != null)
+                {
+                    for (var i = 0; i < DataPointLabels.Length; i++)
+                    {
+                        var label = DataPointLabels[i];
+                        if (!string.IsNullOrEmpty(label))
+                        {
+                            Drawing.Font textFont = new();
```

Comment:
Perhaps this should be customizable? Although I'm happy leaving this simple and adding customization if and when it's necessary.

Reply:
We might as well expose it as a public field `DataPointLabelFont`

I'd make the change myself and merge it now but I don't have write access to the branch 😅